roles/mongodb_linux/molecule/default/tests/test_default.py

```diff
@@ -50,7 +50,7 @@ def test_thp_service(host):
     switches = ["/sys/kernel/mm/transparent_hugepage/enabled",
                 "/sys/kernel/mm/transparent_hugepage/defrag"]
 
-    is_docker = host.file("./is_docker.txt").exists
+    is_docker = host.ansible("setup")["ansible_facts"]["virtualization_type"] == 'docker'
```

Comment:
I'm not sure if there's a performance hit on running this again. I think setup might be called at the beginning for all hosts. It might be faster to use...

https://testinfra.readthedocs.io/en/latest/modules.html#facter

Reply:
Actually, I'm a beginner using `testinfra` so this might as well be clumsy.

Reply:
Isn't facter Puppet specific ? I don't think Ansible supports facter.

Reply:
You're right. It migth be accessible via get_variables https://testinfra.readthedocs.io/en/latest/modules.html#testinfra.modules.ansible.Ansible

Reply:
So do you think something like this would probably work:
```python
host.ansible.get_variables()["ansible_facts"]["virtualization_type"]
```

Reply:
Maybe, you need to try. I wrote these test like 6 months ago. The brain forgets what it doesn't use ,-)